openid-sharedsignals-framework-1_0.md

```diff
@@ -386,6 +387,31 @@ Subject Identifier Format.
     "assertion_id": "_8e8dc5f69a98cc4c1ff3427e5ce34606fd672f91e6"
 }
 
+~~~
+{: #sub-id-ips title="Example: 'ips' Subject Identifier"}
+
+### IP Addresses Subject Identifier Format {#sub-id-ips}
+
+The "IP addresses" Subject Identifier Format specifies an array of IP addresses observed by the Transmitter.
+Subject Identifiers of this format MUST contain the following members:
+
+ips
+
+> REQUIRED. The array of IP addresses of the subject as observed by the Transmitter. The value MUST be in the format of an array of strings, each one of which represents the {{RFC4001}} string representation of an IP address.
+
+
+The "IP addresses" Subject Identifier Format is identified by the name
+"ips".
+
+Below is a non-normative example of Subject Identifier for the "IP addresses"
+Subject Identifier Format.
+
+~~~ json
+{
+    "format": "ips",
+    "ips": ["10.29.37.75", "98.27.134.237"]
```

Comment:
```suggestion
    "ips": ["203.0.113.75", "2001:0db8:e700:37a7:4cd4::b868"]
```
For IPv4 addresses, it is best practice to use allocated example IP ranges (RFC5737). We should also have an IPv6 address in the example (RFC3849).

The IPv6 example uses common shorthand with the double colon. One question is whether we should allow that or require expansion. No opinion.

Reply:
Change of opinion: we should require expanded ones. @timcappalli to find official term (could be "uncompressed", "fully expanded", or "unabbreviated": https://www.ciscopress.com/articles/article.asp?p=2803866)

2024-10-08 call: loose consensus on this